tests/fake_messages/brew-build-group.test.complete.json

```diff
@@ -0,0 +1,74 @@
+{
+  "body": {
+    "msg": {
+      "contact": {
+        "name": "BaseOS CI",
+        "team": "BaseOS",
+        "url": "https://somewhere.com",
+        "docs": "https://somewhere.com/user-documentation",
+        "irc": "#baseosci",
+        "email": "baseos-ci@somewhere.com"
+      },
+      "run": {
+        "url": "https://somewhere.com/job/ci-openstack/4794",
+        "log": "https://somewhere.com/job/ci-openstack/4794/console",
+        "log_raw": "https://somewhere.com/job/ci-openstack/4794/consoleText",
+        "log_stream": "https://somewhere.com/job/ci-openstack/4794/consoleText",
+        "debug": "https://somewhere.com/job/ci-openstack/4794/artifacts/debug.txt",
+        "rebuild": "https://somewhere.com/job/ci-openstack/4794/rebuild/parametrized"
+      },
+      "artifact": {
+        "type": "brew-build-group",
+        "id": "selinux-update-345",
```

Comment:
@hluk please change this to a sha256 hash as with other artifacts in the standard

Reply:
we can do this later ...

Reply:
no action needed

Reply:
I've changed this so some random hash - not sure how exactly will the hash be calculated from the items.

Reply:
ack, that is something which will need to be explained in the standard

Reply:
lgtm